cfgov/unprocessed/apps/prepaid-agreements/css/main.less

```diff
@@ -0,0 +1,119 @@
+@import (reference) '../../../css/main.less';
+@import (less) '../../../css/search.less';
+@import (less) './search-bar.less';
+
+.u-readable-width {
+    max-width: 41.875rem;
+}
+
+.u-break-word {
+    word-break: break-word;
+}
+
+.prepaid-agreements {
+    dt, dd {
+        display: block;
+        margin-left: 0;
+    }
+
+    dt {
+        letter-spacing: 0;
+        font-weight: normal;
+        text-transform: none;
+        font-size: inherit;
+        .heading-4;
+        margin-bottom: unit( @grid_gutter-width / 6 / @base-font-size-px, em );;
+
+        &:last-of-type {
+            margin-bottom: unit( @grid_gutter-width / 6 / @base-font-size-px, em );;
+        }
+    }
+
+    dd + dt {
+        margin-top: unit( ( @grid_gutter-width / 2 ) / @base-font-size-px, em );
+    }
+}
+
+.prepaid-agreements-search {
+    .o-multiselect_label {
+        word-break: break-word;
+    }
+
+    padding-top: 0 !important;
+
+    .filters {
+        &_applied {
+            padding: unit( @grid_gutter-width / 2 / @base-font-size-px, em );
+        }
+        .o-multiselect_choices {
+            display: none;
```

Comment:
Would be more properly set up as a modifier on `o-multiselect`.

Reply:
@anselmbradford That's a good point. I'm working on a couple of multiselect usability tweaks, so I can include a modifier for this in that PR.